### PR TITLE
doc: working_with_nrf: restructuring of nRF52 Series

### PR DESCRIPTION
**Pull request by divipillai**
Renamed Working with to Developing with.
Removed "Developing with" pages.
Created separate pages for the contents
in the developing with guide.

### CODE REVIEW

**Review by kittydepa — APPROVED** [review]
Approving but one question about the order of the docs, on the ``device_guides.rst`` page - were these grouped together in a specific way? If not, could that be addressed after RC1?

File: doc/nrf/device_guides/nrf52/testing.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.. _nrf52_computer_testing:
+
+Testing with multiple nRF52 Series devices
+##########################################
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf52/testing.rst:4]: Do we really need this page? It is repeating a scenario from a sample Testing section, and somewhat repeating https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/gsg_guides/nrf52_gs.html#testing_the_sample .

File: doc/nrf/device_guides/nrf52/testing.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+If you have an nRF52 Series DK with the :ref:`peripheral_uart` sample and either a dongle or second Nordic Semiconductor development kit that supports Bluetooth Low Energy, you can test the sample on your computer.
+Use the Bluetooth Low Energy app in `nRF Connect for Desktop`_ for testing.
+
+To perform the test, :ref:`connect to the nRF52 Series DK <nrf52_gs_connecting>` and complete the following steps:
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf52/testing.rst:9]: If this page is kept, it should rather point to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/test_and_optimize.html and not the getting started guide.

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Comment by divipillai** [comment]
> Approving but one question about the order of the docs, on the `device_guides.rst` page - were these grouped together in a specific way? If not, could that be addressed after RC1?

discussed offline and no changes are required.

**Review by zycz — APPROVED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst [hunk 3]
```diff
@@ -535,45 +535,66 @@ See the following instructions.
...
+FOTA updates
+************
+
+.. include:: /device_guides/nrf52/fota_update.rst
    :start-after: fota_upgrades_intro_start
    :end-before: fota_upgrades_intro_end
 
-.. include:: ../nrf52/developing.rst
+FOTA over Bluetooth Low Energy
+==============================
```
Comment (divipillai) [inline doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst:546]: Please note the header level issue with the "FOTA update sample" section will be fixed in PR https://github.com/nrfconnect/sdk-nrf/pull/15708.

File: doc/nrf/device_guides/nrf52/testing.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/device_guides/nrf52/testing.rst:4]: I don't think it makes sense to keep this page here. There are other samples that have testing scenarios for 2+ development kits, yet here we specifically refer to Peripheral UART. This whole page should rather be a subsection of https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/peripheral_uart/README.html#testing

This whole page was once moved from Infocenter, where it maybe had its purpose, but in the NCS it is redundant.
Re https://github.com/nrfconnect/sdk-nrf/commit/8796a90ba355899c6eafbaf3c68f2488049277b8#diff-5c5fa22b8c1b99fc19b85717fe5a405395e647a4a0f82a7326744c56e35cd4b2

**Review by greg-fer — APPROVED** [review]
I think the nrf52/testing.rst should be removed. Other edits look ok.

**Comment by divipillai** [comment]
> I think the nrf52/testing.rst should be removed. Other edits look ok.

I will cover this in a separate PR. Move the testing section to the Peripheral UART sample.

**Pull request merged**